x-pack/test/observability_ai_assistant_api_integration/common/config.ts

```diff
@@ -45,7 +48,47 @@ export interface CreateTest {
   kbnTestServer: any;
 }
 
-export function createObservabilityAIAssistantAPIConfig({
+function downloadFile(url: string, filePath: string): Promise<void> {
+  return new Promise((resolve) => {
+    const file = fs.createWriteStream(filePath);
+    https.get(url, (response) => {
+      response.pipe(file);
+      file.on('finish', () => {
+        file.close();
+        resolve();
+      });
+    });
+  });
+}
+
+async function downloadElserModel(): Promise<string> {
+  const elserModelDir = path.join('.es', 'cluster-ftr', 'config', 'models');
+  fs.mkdirSync(elserModelDir, { recursive: true });
+  await downloadFile(
+    'https://ml-models.elastic.co/elser_model_2.metadata.json',
+    path.join(elserModelDir, 'elser_model_2.metadata.json')
+  );
+  await downloadFile(
+    'https://ml-models.elastic.co/elser_model_2.pt',
+    path.join(elserModelDir, 'elser_model_2.pt')
+  );
+  await downloadFile(
+    'https://ml-models.elastic.co/elser_model_2.vocab.json',
+    path.join(elserModelDir, 'elser_model_2.vocab.json')
+  );
+  // TODO: Remove again - This is just to show that the file is actually present on the file system
+  // eslint-disable-next-line no-console
+  console.log('Files in elserModelDir:', fs.readdirSync(elserModelDir));
+  // eslint-disable-next-line no-console
+  console.log(
+    'Contents of elser_model_2.metadata.json:',
+    fs.readFileSync(path.join(elserModelDir, 'elser_model_2.metadata.json'), 'utf8')
+  );
+  // TODO: Remove again - end
```

Comment:
@wayneseymour I added the logging the shows that the file is present on the filesystem and can be read.

I just kicked-off another buildkite build and will link and paste the relevant log when the build is finished.

Reply:
https://buildkite.com/elastic/kibana-pull-request/builds/196168#018e1417-bd8a-4e5b-b398-e1e821c0c019

```
_bk;t=1709734258947--- $ node scripts/functional_tests --bail --config x-pack/test/observability_ai_assistant_api_integration/enterprise/config.ts 
_bk;t=1709734293802Files in elserModelDir: [
_bk;t=1709734293802  [32m'elser_model_2.metadata.json'[39m,
_bk;t=1709734293802  [32m'elser_model_2.pt'[39m,
_bk;t=1709734293802  [32m'elser_model_2.vocab.json'[39m
_bk;t=1709734293802]
_bk;t=1709734293802Contents of elser_model_2.metadata.json: {
_bk;t=1709734293802  "packaged_model_id": "elser_model_2",
_bk;t=1709734293802  "minimum_version": "11.0.0",
_bk;t=1709734293802  "size": 438123914,
_bk;t=1709734293802  "sha256": "2e0450a1c598221a919917cbb05d8672aed6c613c028008fedcd696462c81af0",
_bk;t=1709734293802  "description": "Elastic Learned Sparse EncodeR v2",
_bk;t=1709734293802  "model_type": "pytorch",
_bk;t=1709734293802  "tags": [
_bk;t=1709734293802        "elastic"
_bk;t=1709734293802  ],
_bk;t=1709734293802  "inference_config": {
_bk;t=1709734293802    "text_expansion": {
_bk;t=1709734293802      "tokenization": {
_bk;t=1709734293802        "bert": {
_bk;t=1709734293802          "do_lower_case": true,
_bk;t=1709734293802          "with_special_tokens": true,
_bk;t=1709734293802          "max_sequence_length": 512,
_bk;t=1709734293802          "truncate": "first",
_bk;t=1709734293802          "span": -1
_bk;t=1709734293802        }
_bk;t=1709734293802      }
_bk;t=1709734293802    }
_bk;t=1709734293802  },
_bk;t=1709734293802  "vocabulary_file": "elser_model_2.vocab.json"
_bk;t=1709734293802}
```

Reply:
Awesome @reakaleek 

This is good info for us as it not only proves your assumption for the local workstation, but that it holds true within ci.  

Thanks again.